src/MCPClient/lib/clientScripts/bag_with_empty_directories.py

```diff
@@ -83,47 +61,44 @@ def create_directories(base_dir, dir_list):
             pass
 
 
-def bag_with_empty_directories(job, operation, destination, sip_directory, payload_entries, writer, algorithm):
-    """ Run bagit create bag command, and create any empty directories from the SIP. """
+_PAYLOAD_ENTRIES = (
+    "logs/", "objects/", "METS.%SIPUUID%.xml",
+    "README.html", "thumbnails/", "metadata/")
+
+
+def bag_with_empty_directories(job, destination, sip_directory,
+                               sip_uuid, algorithm):
+    """Create BagIt and include any empty directories from the SIP."""
     # Get list of directories in SIP
     dir_list = get_sip_directories(job, sip_directory)
-
     # These are passed to this script as paths relative to their location in
     # the SIP; passing the full SIP location each time has the potential to
     # overflow the 1000-character limit the job's command has in the database,
     # especially with long SIP names.
-    full_paths = [os.path.join(sip_directory, p) for p in payload_entries]
+    full_paths = [os.path.join(sip_directory, p) for p in _PAYLOAD_ENTRIES]
     # Ensure all payload items actually exist
     payload_entries = [e for e in full_paths if os.path.exists(e)]
-
-    # Reconstruct bagit arguments
-    # Goal: bagit <operation> <destination> <flattened payload list> <optional args>
-    bagit_args = [operation, destination]
-    bagit_args.extend(payload_entries)
-    bagit_args.extend(['--writer', writer, '--payloadmanifestalgorithm', algorithm])
-
-    # Run bagit bag creator
-    run_bag(job, bagit_args)
+    os.mkdir(destination)
+    for item in payload_entries:
+        shutil.move(item, destination)
```

Comment:
bagit-java had the ability to create the bag in a different destination while in the Python library the package is created in situ so I'm moving the contents to the actual destination expected by the workflow before the bag is made.